Pipfile

```diff
@@ -19,7 +19,7 @@ pysha3="*"
 requests = "*"
 sqlalchemy = "*"
 apistar = "==0.5.42"
-tzlocal = "==2.0.0b1"
+tzlocal = "==1.5.1"
```

Comment:
Later I'd certainly try `"<2.0.0"`. But it works for now

Reply:
:+1: Agreed